tests/test_ptxboa_streamlit.py

```diff
@@ -16,3 +16,30 @@ def running_app():
 def test_app_smoke(running_app):
     """Test if the app starts up without errors."""
     assert not running_app.exception

```

Comment:
I like testing each tab individually. But then this test here is obsolete, right?